content/learn/getting-started-multi-cloud-gitops.adoc

```diff
@@ -0,0 +1,316 @@
+*** *A specific namespace on the cluster* allows you to choose a specific, single namespace in which to install the Operator. The Operator will only watch and be made available for use in this single namespace.
+
+.. Select *Automatic* or *Manual* approval strategy.
+
+. Click *Install* to make the Operator available to the selected namespaces on this {ocp} cluster.
+
+.Verification
+To confirm that the installation is successful:
+
+. Navigate to the *Operators* → *Installed Operators* page.
+
+. Check that the Operator is installed in the selected namespace and its status is `Succeeded`.
+
+//Include Procedure module here
+[id="create-pattern-instance_{context}"]
+== Creating the Multicloud GitOps instance
+
+.Prerequisites
+The {validated-patterns-op} is successfully installed in the relevant namespace.
+
+.Procedure
+
+. Navigate to the *Operators* → *Installed Operators* page.
+
+. Click the installed *{validated-patterns-op}*.
+
+. Under the *Details* tab, in the *Provided APIs* section, in the
+*Pattern* box, click *Create instance* that displays the *Create Pattern* page.
+
+. On the the *Create Pattern* page, select *Form view* and enter information in the following fields:
+
+** *Name* - A name for the pattern deployment that is used in the projects that you created.
+** *Labels* - Apply any other labels you might need for deploying this pattern.
+** *Cluster Group Name* - Select a cluster group name to identify the type of cluster where this pattern is being deployed. For example, if you are deploying the {ie-pattern}, the cluster group name is `datacenter`. If you are deploying the {mcg-pattern}, the cluster group name is `hub`.
++
+To know the cluster group name for the patterns that you want to deploy, check the relevant pattern-specific requirements.
+. Expand the *Git Config* section to reveal the options and enter the required information.
+. Leave *In Cluster Git Server* unchanged. 
+.. Change the *Target Repo* URL to your forked repository URL. For example, change `+https://github.com/validatedpatterns/<pattern_name>+` to `+https://github.com/<your-git-username>/<pattern-name>+`
+.. Optional: You might need to change the *Target Revision* field. The default value is `HEAD`. However, you can also provide a value for a branch, tag, or commit that you want to deploy. For example, `v2.1`, `main`, or a branch that you created, `my-branch`.
+. Click *Create*.
++
+[NOTE]
+====
+A pop up may throw up an error `"Oh no! Something went wrong`. It is safe to ignore this as the install of the Multicloud GitOps pattern appears unaffected. Watch the ArgoCD instances from the *Hub ArgoCD* UI in the nines menu. They will report progressing/healthy etc. on each of the apps they manage. The *Cluster Argo CD* will have the detailed status on each of the apps defined as such in the clustergroup values file. 
```

Comment:
use "or, and" instead of slash in progressing/healthy
suggestion to rewrite:
A pop-up error may appear with the message, "Oh no! Something went wrong." This can be safely ignored, as the installation of the Multicloud GitOps pattern is unaffected. Monitor the ArgoCD instances through the Hub ArgoCD UI in the nines menu, which will report the status as progressing, healthy, etc. on each managed app. The Cluster ArgoCD provides detailed status on each app, as defined in the clustergroup values file.